[RHACS] ROX-11710 - Make EBPF the default collection method (#47713)
openshift/openshift-docs: modules/secured-cluster-configuration-options-operator.adoc

| Parameter | Description

| `perNode.collector.collection`
| The method for system-level data collection.
The default value is `EBPF`.
Red Hat recommends using `EBPF` for data collection.

> If you select `NoCollection`, you will not see any information about the network activity and the process executions.

Reviewer: ```suggestion
If you select `NoCollection`, then any information about the network activity and the process executions is not available.
```
Just a suggestion. wdyt ?

Author: Thank you @darshan-nagaraj 

🤔 Yeah maybe I could rewrite it as 
```
If you select NoCollection, Collector does not report any information about the 
network activity and the process executions.
```
